patches/colorthief+2.4.0.patch

```diff
@@ -0,0 +1,58 @@
+diff --git a/node_modules/colorthief/src/color-thief-node.d.ts b/node_modules/colorthief/src/color-thief-node.d.ts
```

Comment:
may not steal this for personal projects lmao

Reply:
it's only because of cloudflare LOL